lib/linter.js

```diff
@@ -76,19 +76,13 @@ function parseBooleanConfig(string, comment) {
         if (!name) {
             return;
         }
-        const pos = name.indexOf(":");
 
-        if (pos === -1) {
-            items[name] = {
-                value: false,
-                comment
-            };
-        } else {
-            items[name.slice(0, pos)] = {
-                value: name.slice(pos + 1) === "true",
-                comment
-            };
-        }
+        const [key, value] = name.split(":");
```

Comment:
Non-blocker: To address @platinumazure's readability concerns, we could do something like `const [key, value = false] = name.split(":");`.